res/controllers/Stanton-DJC-4-scripts.js

```diff
@@ -0,0 +1,379 @@
+/**
+ * Stanton DJC4 controller script v1.0 for Mixxx v2.2.3
+ *
+ * Written by Martin Bruset Solberg
+ * Adopted for v2.2.3 by Christoph Zimmermann
+ *
+ * Based on:
+ *  Denon MC2000 script by Esteban Serrano Roloff,
+ *  Denon MC7000 script by OsZ
+ *  Roland DJ-505 script by Jan Holthuis
+ *
+ * TODO:
+ * Pitch range
+ *
+ **/
+
+var djc4 = {};
+
+/////////////////
+// Tweakables. //
+/////////////////
+
+djc4.tempoRange = [0.08, 0.16, 0.5];  // not used yet!
```

Comment:
Please remove this if it's not used anywhere. You can simple re-add it if pitch range is implemented.